### PR TITLE
update helm release annotation inject key

### LINKED ISSUE #4911: The annotation of openelb error
**Describe the Bug**

Request URL:  `/kapis/iam.kubesphere.io/v1alpha2/clusterroles?sortBy=createTime&limit=10&annotation=kubesphere.io%2Fcreator`

openelb should not have this annotation.

image

**Versions Used**
KubeSphere:   `kubespheredev/ks-installer:nightly-20220524`

### PR DESCRIPTION
**Pull request by x893675**
### Which issue(s) this PR fixes:

Fixes #4911

### Special notes for reviewers:
```
Not only openelb chart , the helm chart that contains the clusterrole template will have the same problem
```

### Does this PR introduced a user-facing change?

```release-note
None
```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/controller/openpitrix/helmrelease/helm_release_controller.go [hunk 1]
```diff
@@ -339,7 +338,6 @@ func (r *ReconcileHelmRelease) createOrUpgradeHelmRelease(rls *v1alpha1.HelmRele
 
 	// If clusterConfig is empty, this application will be installed in current host.
 	hw := helmwrapper.NewHelmWrapper(clusterConfig, rls.GetRlsNamespace(), rls.Spec.Name,
-		helmwrapper.SetAnnotations(map[string]string{constants.CreatorAnnotationKey: rls.GetCreator()}),
```
Comment (xyz-li) [inline pkg/controller/openpitrix/helmrelease/helm_release_controller.go:342]: We can't simply remove this field, front end needs this value to display the creator of the app, you can change the key to apps.kubesphere.io/creator.

**Comment by x893675** [comment]
fixed.. @xyz-li

**Comment by xyz-li** [comment]
Please change the key in the function `GetCreator` too.
https://github.com/kubesphere/kubesphere/blob/587dbe723bf80f5c96ab1d49aa92c4c2311a8bef/staging/src/kubesphere.io/api/application/v1alpha1/helmrelease_types.go#L119-L121

**Comment by xyz-li** [comment]
There 5 `GetCreator` functions in dir staging/src/kubesphere.io/api/application/v1alpha1. You need to change them all.

**Comment by x893675** [comment]
QQ20220811-131004@2x

front end needs update too

**Comment by x893675** [comment]
@xyz-li

**Comment by xyz-li** [comment]
> QQ20220811-131004@2x
> 
> when resource is rendered by helm release, front end needs use new annotation key

This is a dilemma.

**Comment by xyz-li** [comment]
/hold

We need to find a better way to deal with this annotation.

**Pull request closed without merging**